src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs

```diff
@@ -66,6 +66,7 @@ public static partial class PlatformDetection
         public static bool IsNotArmProcess { get { throw null; } }
         public static bool IsNotFedoraOrRedHatFamily { get { throw null; } }
         public static bool IsNotMacOsHighSierraOrHigher { get { throw null; } }
+        public static bool IsNotNetNative { get { throw null; } }
```

Comment:
What's difference between IsNotNetNative  and IsNotNetNativeRunningAsConsoleApp ?

Reply:
Here's a comment:
```
        // Officially, .Net Native only supports processes running in an AppContainer. However, the majority of tests still work fine
        // in a normal Win32 process and we often do so as running in an AppContainer imposes a substantial tax in debuggability
        // and investigatability. This predicate is used in ConditionalFacts to disable the specific tests that really need to be
        // running in AppContainer when running on .NetNative.
        public static bool IsNotNetNativeRunningAsConsoleApp => !(IsNetNative && !IsInAppContainer);
```

Basically we just want the inverted version of `IsNetNative`

Reply:
If `IsNetNative` exists, why do we need another getter just for the negation? Why not just type `!IsNetNative`? What am a I missing?

Reply:
Yep - missing something 😄 Thanks!